### PR TITLE
[DPM-823] convert bonus

### PR DESCRIPTION
**Pull request by alrxy**

### CODE REVIEW

**Comment by alrxy** [comment]
check after #43

File: bonus.go [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+	if err != nil {
+		return nil, err
+	}
+
+	return playerStats, nil
+}
+
+func (app *App) convertBonus(player string, force bool) error {
+	if !force {
+		meetRequirements, err := app.meetRequirements(player)
```
Comment (justefg) [inline bonus.go]: can we change the signature to 
`PlayerMeetsRequirements(player string, bcAPI *eos.API) (bool, error)`?
not clear what `app.meetRequiremens` means without going into implementation

File: blockchain.go [hunk 2]
```diff
@@ -154,11 +154,46 @@ func SendPackedTrxWithRetries(bcAPI *eos.API, packedTrx *eos.PackedTransaction,
 			if apiErr, ok := e.(eos.APIError); ok {
 				// if error is duplicate trx assume as OK
 				if apiErr.Code == EosInternalErrorCode && apiErr.ErrorStruct.Code == EosInternalDuplicateErrorCode {
-					log.Debug().Msgf("Got duplicate trx error, assuming as OK, trx_id: %s", trxID)
+					log.Debug().Msgf("Got duplicateK trx error, assuming as OK, trx_id: %s", trxID)
```
Comment (justefg) [inline blockchain.go]: K?)

File: bonus.go (hunk 1 above)
Comment (alrxy) [reply bonus.go]: ok

File: blockchain.go (hunk 2 above)
Comment (alrxy) [reply blockchain.go]: k)

**Review by justefg — APPROVED** [review]

**Review by justefg — APPROVED** [review]

**Pull request merged**